app/controllers/albums.js

```diff
@@ -11,3 +11,9 @@ exports.getAlbumPhotos = (req, res, next) =>
     .getPhotos({ albumId: req.params.id, ...req.query })
     .then(photoData => res.send(photoData))
     .catch(error => next(error));
+
+exports.buyAlbum = (req, res, next) =>
+  albumsService
+    .buyAlbum({ albumId: req.params.id, userId: req.body.user_id })
+    .then(response => res.send(response))
+    .catch(error => next(error));
```

Comment:
Is this PR of buy album also?

Reply:
During the merge I must have moved things around when solving conflicts